src/genophenocorr/preprocessing/_vv.py

```diff
@@ -155,17 +157,24 @@ def _parse_tx(tx: typing.Union[str, TranscriptInfoAware]) -> str:
             raise ValueError(f'Expected a `str` or `TranscriptInfoAware` but got {type(tx)}: {tx}')
 
     def parse_response(self, tx_id: str, response) -> TranscriptCoordinates:
-        if len(response) != 1:
-            self._logger.warning('Response has %s!=1 items. Choosing the first', len(response))
-
-        item = response[0]
-        if 'requested_symbol' not in item or item['requested_symbol'] != tx_id:
-            raise ValueError(
-                f'Could not find {tx_id} in the `requested_symbol` field in the response from Variant Validator API'
-            )
-        if 'transcripts' not in item:
+        if not isinstance(response, list):
```

Comment:
The aim of this function is to process the response into `TranscriptCoordinates` or to raise a `ValueError` if there are any issues with the input.

I wrote few tests for you to play with the function. Please figure out what you are after and remove the print statements at the end so that we do not litter STDERR :)

Reply:
@ielis I should have left this as draft. The problem is that response from VariantValidator is not always exactly what this function expects, and I I added the print statement to get better feedback than from throwing an error. Testing is hard because we do not have all inputs but I will try to get something reasonable

Reply:
Hm.. :/ If you're seeing errors, then test suite cannot be comprehensive, I only tested ~4 transcripts :)

Please let me know when you receive a weird response from VV, I'll update the tests & fix the code.

Reply:
Working with this, I think the PR is ready to be merged so we can move ahead. I cannot reproduce the bud I was seeing anymore.